frontend/src/container/ErrorDetails/index.tsx

```diff
@@ -161,7 +163,9 @@ function ErrorDetails(props: ErrorDetailsProps): JSX.Element {
 			</DashedContainer>
 
 			<Typography.Title level={4}>{t('stack_trace')}</Typography.Title>
-			<Editor onChange={(): void => {}} value={stackTraceValue} readOnly />
+			<div className="errorContainer">
```

Comment:
Let's keep the className in Kebab Case throughout the application. 
```suggestion
			<div className="error-container">
```